packages/trader/test/index.ts

```diff
@@ -1,23 +1,187 @@
 import { run } from "../src/index";
```

Comment:
This set of tests correctly validates that the `index` can start up with no error. In future this should include re-try logic tests as well as some others.